src/main/java/com/ithaca/BookBudsApplication.java

```diff
@@ -18,6 +21,8 @@ public FilterRegistrationBean jwtFilter() {
 		registrationBean.addUrlPatterns("/users/account");
 		registrationBean.addUrlPatterns("/messages");
```

Comment:
This is about messages but I just noticed in app.js in the service controller its referred to as "/users/messages", not just "/messages" does that need to be the same in this spot and in the app.js?